tests/API/IEXTest.cpp

```diff
@@ -0,0 +1,45 @@
+#define CATCH_CONFIG_MAIN
+#include <catch2/catch.hpp>
+#include <catch2/trompeloeil.hpp>
+
+#include "../../src/API/IEX.h"
+
+class IEX_Mock : public API::IEX
+{
+public:
+    MAKE_MOCK2(getParsedJson, json(const std::string, const bool), override);
```

Comment:
:+1: Nice, the ability to use the `override` keyword is a big advantage of trompeloeil over gmock